src/client/auth/oidc.rs

```diff
@@ -123,8 +125,10 @@ pub struct CallbackInner {
 #[derive(Debug, Clone)]
 pub struct Cache {
     idp_server_info: Option<IdpServerInfo>,
-    refresh_token: Option<String>,
-    access_token: Option<String>,
+    // pub(crate) for spec tests
```

Comment:
Suggest adding a test-only module to expose these values like we do here for the AWS tests: https://github.com/mongodb/mongo-rust-driver/blob/ec3a644d5eb89f491ac452951c0854f6477fdfe1/src/client/auth/aws.rs#L553

that way we can keep these values non-`pub(crate)` which is nice for encapsulation